Userland/Applications/FileManager/main.cpp

```diff
@@ -1078,7 +1096,14 @@ ErrorOr<int> run_in_windowed_mode(DeprecatedString const& initial_location, Depr
         auto* bitmap = icon.bitmap_for_size(16);
         window->set_icon(bitmap);
 
-        window->set_title(DeprecatedString::formatted("{} - File Manager", new_path));
+        DeprecatedString title = new_path;
```

Comment:
not strictly an issue, but… 😉

— it seems that this new part is something that can be more or less extracted (i.e. title path computation looks very nice; then you could even go with immutability for the price of a [potential] method call), esp. that thereʼs a code above, you do roughly the same thing? 🤔

Reply:
The computation is now extracted to a helper function; I wanted to make it a method, but who gets it? `DeprecatedString` is, well, deprecated, and converting to `String` is enough work to get its own PR.